synapse/app/homeserver.py

```diff
@@ -427,6 +428,13 @@ def phone_stats_home():
 
         daily_sent_messages = yield hs.get_datastore().count_daily_sent_messages()
         stats["daily_sent_messages"] = daily_sent_messages
+        if len(stats_process) > 0:
+            stats["memory_rss"] = 0
+            stats["cpu_average"] = 0
+            for process in stats_process:
+                with process.oneshot():
```

Comment:
This doesn't exist in older versions of psutil, alas. We have theoretically support any version of psutil >= 2. and in practice I think the debian package (for jessie) relies on being able to use v4.x

Reply:
:+1: